File: OLD Repos/Linear Algebra/Linear Operations Foundation/Linear Change.py
#### Linear Change

## From x in [x_min,x_max] to y in [y_min,y_max]
# y=y_min+(x-x_min)(y_max-y_min)/(x_max-x_min)

import numpy as np

X = np.arange(0,11,1)           # array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10])
print("X:")
for n in np.arange(0,11,1):
    print(X[n])

print("Y[n]=4*X[n]+20:")
Y = 4*X+10
for n in np.arange(0,11,1):
    print(Y[n])

File: Appendix Linear Algebra/Linear Operations Foundation/Linear Change.jl
#### Linear Change

## From x in [x_min,x_max] to y in [y_min,y_max]
# y=y_min+(x-x_min)(y_max-y_min)/(x_max-x_min)

X = range(0,stop=10,step=1)
println("X:")
for n in 1:11
    print(X[n],"\t")
end
println()

println("Y[n]=4*X[n]+20:")
Y = [4*X[n]+10 for n in 1:11]
for n in 1:11
    print(Y[n],"\t")
end
println()
